website/src/content/api-reference/commercetools-frontend-application-components.mdx

```diff
@@ -209,109 +151,64 @@ const EmailForm = () => {
-| `getParentSelector`       | `function`         |    -     | -                        | The function should return an HTML element that will be used as the parent container to hold the modal DOM tree. If no function is provided, it's expected that an HTML element with the `id="parent-container"` is present in the DOM. In the `NODE_ENV=test` environment, the default HTML element is `body`.                                                                                                                                                                                                            |
-| `shouldDelayOnClose`      | `bool`             |    -     | `true`                   | Sets whether the ModalPage should delay calling its `onClose` function to allow the closing animation time to finish. This can be turned off if the developer is controlling the ModalPage only through the `isOpen` prop, and not abruptly mounting/unmounting it or one of its parent elements. You might also want to turn this off if you need to display a prompt (for example to save changes) on the ModalPage before navigating out of it, as this option makes the Modal close itself before `onClose` is called. |
-| `afterOpenStyles`         | `string` or `object`  |    -  | -                        | Overwrite the default styles of `afterOpen`. You can pass a "class name" or a CSS-in-JS style object. This should be used only in cases the default styles are causing some layout issues.                                                                                                                                                                                                                                                                                                                                 |
 
-## FormModalPage
+## CustomFormMainPage
 
-<Playground urlPath="form-modal-page" />
+<Playground urlPath="custom-form-main-page" />
 
-Form Modal pages are controlled components used to render a page with a form or something that requires user input, similar to `InfoModalPage` but semantically with a different role. The header includes buttons to control the submission or cancellation of the form. These buttons can be overwritten with custom controls by using the `<CustomFormModalPage>` component.
+Custom Form Main page is a variation of the `<FormMainPage>` that allow passing custom control elements via formControls. This is useful in case the main page needs different control elements than the default ones (primary and secondary button).
```

Comment:
```suggestion
Custom Form Main page is a variation of the `<FormMainPage>` that allows passing custom control elements via formControls. This is useful in case the main page needs different control elements than the default ones (primary and secondary buttons).
```